### DISCUSSION TITLE
Why did I get an outdated html string

### DISCUSSION THREAD

**Discussion by rieval** [Milkdown/milkdown#775] [Q&A]
code:
```typescript
const { editor, loading, getInstance } = useEditor((root, _renderVue) => {
  const editor = Editor.make()
    .config((ctx) => {
      ctx.set(rootCtx, root)
    })
    .config((ctx) => {
      ctx
        .get(listenerCtx)
        .mounted((ctx) => {
          editorRef.value = ctx.get(editorCtx)
          mounted.value = true
        })
        .updated((ctx, _doc, _prevDoc) => {
          const html = getHTML()
          console.log(html(ctx)) // print result here
        })
// snip
```

result:
![image](https://user-images.githubusercontent.com/22883406/196920851-17fcf1c8-4ff5-4f31-84b2-33678addac4e.png)

**Comment by Saul-Mirone** [comment]
Does this only happen for Vue?

**Reply by rieval** [reply]
> Does this only happen for Vue?

I'm not sure, it was just tested for Vue.
Maybe it looks like this issue: https://github.com/Saul-Mirone/milkdown/discussions/466#discussioncomment-2734683

**Reply by LBiddiscombe** [reply]
Hi. Same in svelte for me, calling it the exact same way and getHTML is always 1 update behind.
